sdk/cognitiveservices/pom.xml

```diff
@@ -296,23 +303,23 @@
     </pluginManagement>
   </build>
   <modules>
-    <module>./language/bingspellcheck</module>
-    <module>./language/luis/authoring</module>
-    <module>./language/luis/runtime</module>
-    <module>./language/textanalytics</module>
-    <module>./search/bingautosuggest</module>
-    <module>./search/bingcustomimagesearch</module>
-    <module>./search/bingcustomsearch</module>
-    <module>./search/bingentitysearch</module>
-    <module>./search/bingimagesearch</module>
-    <module>./search/bingnewssearch</module>
-    <module>./search/bingvideosearch</module>
-    <module>./search/bingvisualsearch</module>
-    <module>./search/bingwebsearch</module>
-    <module>./vision/computervision</module>
-    <module>./vision/contentmoderator</module>
-    <module>./vision/customvision/prediction</module>
-    <module>./vision/customvision/training</module>
-    <module>./vision/faceapi</module>
+    <module>microsoft-azure-cognitiveservices-spellcheck</module>
+    <module>microsoft-azure-cognitiveservices-luis-authoring</module>
+    <module>microsoft-azure-cognitiveservices-luis-runtime</module>
+    <module>microsoft-azure-cognitiveservices-textanalytics</module>
+    <module>microsoft-azure-cognitiveservices-autosuggest</module>
+    <module>microsoft-azure-cognitiveservices-customimagesearch</module>
+    <module>microsoft-azure-cognitiveservices-customsearch</module>
+    <module>microsoft-azure-cognitiveservices-entitysearch</module>
+    <module>microsoft-azure-cognitiveservices-imagesearch</module>
+    <module>microsoft-azure-cognitiveservices-newssearch</module>
+    <module>microsoft-azure-cognitiveservices-videosearch</module>
+    <module>microsoft-azure-cognitiveservices-visualsearch</module>
+    <module>microsoft-azure-cognitiveservices-websearch</module>
+    <module>microsoft-azure-cognitiveservices-computervision</module>
+    <module>microsoft-azure-cognitiveservices-contentmoderator</module>
+    <module>microsoft-azure-cognitiveservices-customvision-prediction</module>
+    <module>microsoft-azure-cognitiveservices-customvision-training</module>
+    <module>microsoft-azure-cognitiveservices-faceapi</module>
```

Comment:
Just confirming that this is the desired structure after moving to /sdk - that we flatten subdirectories of 'language', 'search', and 'vision' (and subdirectories in them) all into the root directory? Some of these names are getting so long that on my machine they are not readable (look here, especially try to find the custom vision training module: https://github.com/mitchdenny/azure-sdk-for-java/tree/move-cognitive-services-to-sdk/sdk/cognitiveservices) // cc @weshaggard

Reply:
This is deliberate. Trying to enforce consistency. In the case of cognitive services we might end up seperating each sub-service into a different folder under ```sdk/```.

Reply:
We hit long path issues for this one in .NET repo as well and to help we removed the redundancy in the name so in this case we could remove "microsoft-azure-cognitiveservices" from each of these names. However once we have track2 in this directory we like will have to add at least part of that name back into the directory.

Reply:
So I've renamed the modules to ```ms-azure-cs-{thing}``` to give us some path space back. The issue wasn't with the sources itself, it was actually the targets folder - so this buys us some headroom.